meetings/2025-04-03.md

```diff
@@ -0,0 +1,72 @@
+| Attendees:      | Andreu Botella (Igalia), Ethan Arrowood (HarperDB, Invited Expert), Dean Srebnik (Andromeda, Invited Expert), Owen Buckley (Project Evergreen, Invited Expert) |
+| Secretary:      |                                                                                                                                 Aki Braun (Ecma International) |
+ 
+## 1     Welcome, opening and meeting logistics
+
+**Andreu Botella**, vice chair of TC55, began the teleconference and welcomed attendees. All attendees were informed that the meeting was recorded and transcribed by Google Gemini for the purpose of note-taking.
+
+## 2     Approval of the minutes of the virtual meeting of20 March 2025
+
+The [pull request](https://github.com/wintercg/admin/pull/110) of the minutes from the 4th meeting of TC55 were acknowledged and approved.
+
+## 3   Review of the agenda
+
+The agenda was created from the comments on **[the meeting’s associated GitHub issue](https://github.com/wintercg/admin/issues/111)**, as is the standard practice for TC55. It was proposed and approved at the start of the meeting.
+
+* Continue WPT review
+  * `DOMException`
+  * `Event`
+  * `EventTarget`
+* `Fetch`-related APIs (`Header`, `Request`, `Response`)
+* WASM
+* `globalThis`: `navigator`, `performance.now` `queueMicrotasks`, `setInterval`/`setTimeout`, `structuredClone`, `atob`/`btoa`
+
+## 4     Discussions
+
+### 4.1 Web Platform Test review
+
+Andreu Botella picked up the WPT review where Luca Casonato left off at the 4th meeting of TC55. Andreu, with the support of the attendees, concluded that this review was best facilitated by Luca, who has been doing the bulk of the work between meetings.
+
+### 4.2 Fetch Specification review
+
+The committee continued on working through aspects of Fetch which will or will not be implemented by server-side runtimes.
+
+#### 4.2.1 HTTP Extensions Headers (continued from previous meetings)
+
+* context sniffing—the nosniff-related mimetype restrictions in the fetch API for browsers should not have any impact on a server-side implementation
+* Cross-origin restrictions are not relevant to a server environment and will be ignored
+* Sec-Purpose header is related to browser prefetch and will be ignored
+ 
+#### 4.2.2 Accept Headers
+
+* accept-language is not relevant without knowing the destination
+ 
+#### 4.2.3 Main Fetch Algorithm
+
+Andreu added [a comment](https://github.com/wintercg/fetch-workstream/issues/1#issuecomment-2776237976) to [the tracking issue](https://github.com/wintercg/fetch-workstream/issues/1) on the [fetch-workstream](https://github.com/wintercg/fetch-workstream/) repo to summarize review of the “main” fetch algorithm. Important open questions are primarily security-related; referrer, CSP, HSTS, CORS, “bad” ports, etc. The ports, specifically, are addressed in [a sub-issue](https://github.com/wintercg/fetch-workstream/issues/10) on the repo.
```

Comment:
```suggestion
Andreu added [a comment](https://github.com/wintercg/fetch-workstream/issues/1#issuecomment-2776237976) to [the tracking issue](https://github.com/wintercg/fetch-workstream/issues/1) on the [fetch-workstream](https://github.com/wintercg/fetch-workstream/) repo to summarize review of the “main” fetch algorithm. There are some open issues such as figuring out what exactly to do about HSTS and HTTPS redirection, or what spec changes would be needed to avoid CORS blocking. An important open issue which is security-relevant is blocking “bad” ports, which is addressed in [a sub-issue](https://github.com/wintercg/fetch-workstream/issues/10) on the repo.
```